events/message.go

```diff
@@ -91,9 +95,13 @@ type EventMessage struct {
 	Source string `json:"source"`
 	// Timestamp is the time representing when the message was created
 	Timestamp time.Time `json:"timestamp"`
+	// TraceContext is a map of values used for OpenTelemetry context propagation.
+	TraceContext map[string]string `json:"traceContext"`
```

Comment:
Is there a reason to use the underlying type of `propogation.MapCarrier` instead of that type directly?

Reply:
Yeah, my thinking is that a service using this type should not be free to just write arbitrary keys, or should need to pass through the interface the carrier provides. :ooking at the otel SDK again, `propogation.MapCarrier` is just one implementation, so I think this is the best way?